src/lazyload-image.directive.ts

```diff
@@ -25,14 +25,14 @@ export class LazyLoadImageDirective implements OnChanges, AfterContentInit, OnDe
   private ngZone: NgZone;
   private scrollSubscription?: Subscription;
   private hooks: HookSet<any>;
-  private platformId: Object;
+  private platformId: object;
 
-  constructor(el: ElementRef, ngZone: NgZone, @Inject(PLATFORM_ID) platformId: Object, @Optional() @Inject('options') options?: ModuleOptions) {
+  constructor(el: ElementRef, ngZone: NgZone, @Inject(PLATFORM_ID) platformId: object, @Optional() @Inject('options') options?: ModuleOptions) {
```

Comment:
It should be `Object` here. `@Inject(PLATFORM_ID)` will resolve a `Object` and `isPlatformServer` accepts an `Object`. This row also makes the build to fail right now with the following error: `Metadata collected contains an error that will be reported at runtime: Expression form not supported` but if I change it to `Object` it works fine.

Reply:
ahah, had this in mind: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Didn't realise it was needed here. Will fix throughout